spec/ref_arch_setup/bolt_helper_spec.rb

```diff
@@ -74,25 +76,16 @@
 
   describe "run_task_with_bolt" do
     before do
-      @env_vars = "VAR1=1 "
-      @env_vars << "VAR2=2"
       @expected_command = "bolt task run #{task} VAR1=1 VAR2=2 --modulepath "\
       "#{RefArchSetup::RAS_MODULE_PATH} --nodes #{nodes}"
-      @task_params = { task: task, env_vars: @env_vars, nodes: nodes }
     end
 
     context "when bolt works and returns true" do
       expected_output = "All Good"
       expected_status = 0
-      it "returns true" do
-        expect(RefArchSetup::BoltHelper).to receive(:`)\
-          .with(@expected_command).and_return(expected_output)
-        `(exit #{expected_status})`
-        expect($?).to receive(:success?).and_return(true) # rubocop:disable Style/SpecialGlobalVars
-        expect(RefArchSetup::BoltHelper.run_task_with_bolt(@task_params)).to eq(true)
-      end
-
-      it "outputs informative messages" do
+      it "returns true and outputs informative messages" do
+        expect(RefArchSetup::BoltHelper).to receive(:params_to_string) \
+          .with(params).and_return(params_str)
```

Comment:
I had broken out some of the validations into separate 'it' sections based on feedback from @er0ck

Reply:
lol, I took it out thinking Eric always likes faster, and less tests are faster. :)   We will have to consider if we want to split out other similar tests in our test base.

Reply:
oh, and I put it back in